src/Rules/Properties/DefaultValueTypesAssignedToPropertiesRule.php

```diff
@@ -39,7 +39,7 @@ public function processNode(Node $node, Scope $scope): array
 		$propertyReflection = $classReflection->getNativeProperty($node->getName());
 		$propertyType = $propertyReflection->getWritableType();
 		if ($propertyReflection->getNativeType() instanceof MixedType) {
-			if ($default instanceof Node\Expr\ConstFetch && (string) $default->name === 'null') {
+			if ($default instanceof Node\Expr\ConstFetch && $default->name->toLowerString() === 'null') {
```

Comment:
btw: the origin reported mentioned he expected 2 errors instead. since we already have this special case for native `mixed` I went with 0 errors for consistency.